Assets/PatchKit/UnityEditorExtension/Editor/UI/SafeBuildAndUploadScreen.cs

```diff
@@ -219,7 +230,8 @@ private void DrawContent()
                 EditorGUILayout.Toggle(_overwriteDraftVersion);
         }
         EditorGUILayout.EndHorizontal();
-
+        
+        bool isUploadPermitted = false;

```

Comment:
`isUploadAllowed` would be more easier to understand.

Reply:
How about `canUpload` ?

Reply:
Yup, it's also good.